controllers/components/modelregistry/modelregistry_controller_actions.go

```diff
@@ -58,6 +58,14 @@ func initialize(ctx context.Context, rr *odhtypes.ReconciliationRequest) error {
 		return fmt.Errorf("resource instance %v is not a componentApi.ModelRegistry)", rr.Instance)
 	}
 
+	// update registries namespace in manifests
+	mi := baseManifestInfo(BaseManifestsSourcePath)
+	if err := odhdeploy.ApplyParams(mi.String(), nil, map[string]string{
```

Comment:
i am fine with the change
nit: do we need to create variable mi here?
if so, b`aseManifestInfo(BaseManifestsSourcePath)` can be replaced with it (line70)
or just use it directy in line63

Reply:
Good nit, got me thinking about what happens with devflag manifests later.

Reply:
Can't add comments on unchanged lines. Do we need to odhdeploy.ApplyParams() again inside the if block after line 99?

Reply:
I pulled it out into another action `customizeManifests`, called after `initialize`, that way devflags will be handled correctly.